### PR TITLE
Banana & peel resprite.

### PR DESCRIPTION
**Pull request by Krysonism**
## About The Pull Request
This PR resprites the bananas and peels.

## Why It's Good For The Game
This makes bananas and peels look a bit prettier.

old vs new peels(3 variants).
peel_comp

old vs new bananas

banana_comp

## Changelog
:cl:
imageadd: Bananas & peels have been resprited.
/:cl:

### CODE REVIEW

**Comment by Thunder12345** [comment]
Soulless

**Comment by ArcaneMusic** [comment]
Looks great.
Also, stray white pixel in the bluenana animation.

**Comment by Tlaltecuhtli** [comment]
give mime bananas some transparency

**Comment by ShibaInuLord** [comment]
muh soul

**Comment by Tennessee116** [comment]
I'd say the banana is a little too curved, other than that its a good sprite.

**Comment by Qustinnus** [comment]
the variant where you only see 2 of the sides looks really flat because of it, could we maybe see the third side in the background or something?

**Comment by ExcessiveUseOfCobblestone** [comment]
variant 1 (row 2) retains the soul so it is the best

**Comment by BeloneX** [comment]
Soul.....................................

**Comment by D4C-420** [comment]
Da soul

File: code/modules/hydroponics/grown/banana.dm [hunk 1]
```diff
@@ -62,6 +62,14 @@
 	throw_speed = 3
 	throw_range = 7
 
+/obj/item/grown/bananapeel/Initialize()
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/hydroponics/grown/banana.dm:65]:
```suggestion
/obj/item/grown/bananapeel/Initialize(mapload)
```

**Pull request merged**